### DISCUSSION TITLE
Welcome to intrepidbot Discussions!

### DISCUSSION THREAD

**Discussion by intrepidbird** [intrepidbird/gauss#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

I'm the creator and owner of IntrepidBot Mathematics, and I am happy to set up discussions!

**Comment by Prathamesh-007** [comment]
Hello! After going through the main.py file, I saw some functions that can be optimized and some new functions such as nPr and nCr can be added. Is it ok if I do that?

**Reply by intrepidbird** [reply]
Yes! Feel free!
